openeogeotrellis/logs.py

```diff
@@ -34,6 +38,15 @@ def _elasticsearch_logs(job_id: str, search_after: Optional[list]) -> Iterable[d
                             "filter": [
                                 {"term": {"job_id": job_id}},
                                 {"terms": {"tags": ES_TAGS}},
+                                {
+                                    "range": {
+                                        "@timestamp": {
+                                            "format": "strict_date_optional_time",
+                                            "gte": rfc3339.datetime(create_time),
+                                            "lte": "now",
```

Comment:
I don't think the `"lte"` is needed.

Reply:
Indeed. I left it out now. It should work the same.